Patch104pZH/GameFilesEdited/Data/INI/ObjectCreationList.ini

```
@@ -6685,7 +6685,7 @@ End
 ObjectCreationList OCL_ChinaTankGattlingDebris
   CreateObject
     ObjectNames = DeadChinaGattlingTankHulk
-    Offset      = X:-6.62 Y:0.0 Z:0.0
+    Offset      = X:0.0 Y:0.0 Z:0.0 ; Patch104p @bugfix from X:-6.62 Y:0.0 Z:0.0
```

```suggestion
    Offset      = X:0.0 Y:0.0 Z:0.0 ; Patch104p @bugfix Fix offset.
```
No need to keep this value. It is from copy paste and does not apply to these models.

---

we could also remove the entire offset value.

---

Idk if they are optinonal. They appear in any other wreck OCL, even if all values are 0.0 x3.

---

Fixed